Sources/MDFInternationalization.h

```diff
@@ -16,9 +16,9 @@
 
 #import <UIKit/UIKit.h>
 
-#import <MDFInternationalization/MDFRTL.h>
-#import <MDFInternationalization/UIImage+MaterialRTL.h>
-#import <MDFInternationalization/UIView+MaterialRTL.h>
+#import "MDFRTL.h"
+#import "UIImage+MaterialRTL.h"
+#import "UIView+MaterialRTL.h"
```

Comment:
We are following Apple's guidelines for frameworks here.
Is this change required for kokoro?

Reply:
Yes it's unfortunately the lowest common denominator for building succesfully across CocoaPods/Xcode/Bazel.